airbyte-cdk/python/unit_tests/sources/file_based/file_types/test_parquet_parser.py

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+import math
+from typing import Any, Mapping
+
+import pyarrow as pa
+import pytest
+from airbyte_cdk.sources.file_based.file_types import ParquetParser
+from pyarrow import Scalar
+
+
+@pytest.mark.parametrize(
+    "parquet_type, expected_type",
+    [
+        pytest.param(pa.bool_(), {"type": "boolean"}, id="test_parquet_bool"),
+        pytest.param(pa.int8(), {"type": "integer"}, id="test_parquet_int8"),
+        pytest.param(pa.int16(), {"type": "integer"}, id="test_parquet_int16"),
+        pytest.param(pa.int32(), {"type": "integer"}, id="test_parquet_int32"),
+        pytest.param(pa.uint8(), {"type": "integer"}, id="test_parquet_uint8"),
```

Comment:
Should this and the other test include `null`, `int64` and `uint64`, `fixed_size_binary`, and/or `map`?

Reply:
yes, done